dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -949,6 +950,81 @@ def run_generate_constraints_in_parallel(
     )
 
 
+@release_management.command(
+    name="tag-providers",
+    help="Generates tags for airflow provider releases.",
+)
+@click.option(
+    "--clean-local-tags",
```

Comment:
NIT: 

```suggestion
    "--clean-local-tags/--no-clean-local-tags",
```

This will allow you to toggle it to false by specifying `--no-clean-local-tags`.

Reply:
https://click.palletsprojects.com/en/8.1.x/options/#boolean-flags

Reply:
Added this.